### PR TITLE
Update add_kubernetes_metadata.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
jenkins, run the tests please

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 The `add_kubernetes_metadata` processor annotates each event with relevant
 metadata based on which Kubernetes pod the event originated from.
-At startup it detects an `in_cluster` environment and caches the
+At startup, it detects an `in_cluster` environment and caches the
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:10]: A comma is OK, but the general rule is to add a comma after an introductory phrase of 4 or more words.

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 2]
```diff
@@ -19,22 +19,22 @@ Each event is annotated with:
 * Namespace
 * Labels
 
-In addition Node's metadata and Namespace's metadata are being added in Pod's metadata.
+In addition, Node's metadata and Namespace's metadata are being added in Pod's metadata.
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:22]:
```suggestion
In addition, the node and namespace metadata is added to the pod metadata.
```

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 3]
```diff
@@ -19,22 +19,22 @@ Each event is annotated with:
...
 
-The `add_kubernetes_metadata` processor has two basic building blocks which are:
+The `add_kubernetes_metadata` processor has two basic building blocks:
 
 * Indexers
 * Matchers
 
 Indexers use pods metadata to create unique identifiers for each one of the
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:29]:
```suggestion
Indexers use pod metadata to create unique identifiers for each one of the
```
```suggestion
Indexers use pods metadata to create unique identifiers for each one of the
```

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 4]
```diff
@@ -118,9 +118,9 @@ mode.
 `namespace`:: (Optional) Select the namespace from which to collect the
 metadata. If it is not set, the processor collects metadata from all namespaces.
 It is unset by default.
-`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and Namespace.
+`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and namespace.
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:121]: Why is Node capitalized? It should be lowercase, no? See the [K8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/).

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 5]
```diff
@@ -118,9 +118,9 @@ mode.
...
  `add_resource_metadata` can be done for `node` or `namespace`. By default all labels will be included
-while annotations are not added by default. This settings are useful when labels' and annotations'
+while annotations are not added by default. These settings are useful when labels' and annotations'
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:123]: Should labels and annotations be possessive here?

**Review by dedemorton — CHANGES_REQUESTED** [review]
Thanks for the contribution! I think you need to make another pass at this topic because you have missed some sentences that need attention. I've marked a few, but there are others. Thanks!

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 6]
```diff
@@ -118,10 +118,10 @@ mode.
 `namespace`:: (Optional) Select the namespace from which to collect the
 metadata. If it is not set, the processor collects metadata from all namespaces.
 It is unset by default.
-`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and Namespace.
+`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from node and namespace.
  `add_resource_metadata` can be done for `node` or `namespace`. By default all labels will be included
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:122]:
```suggestion
 `add_resource_metadata` can be done for `node` or `namespace`. By default all labels are included
```

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 7]
```diff
@@ -136,7 +136,5 @@ Example:
 -------------------------------------------------------------------------------------
 `kube_config`:: (Optional) Use given config file as configuration for Kubernetes
 client. It defaults to `KUBECONFIG` environment variable if present.
-`default_indexers.enabled`:: (Optional) Enable/Disable default pod indexers, in
-case you want to specify your own.
-`default_matchers.enabled`:: (Optional) Enable/Disable default pod matchers, in
-case you want to specify your own.
+`default_indexers.enabled`:: (Optional) Enable or disable default pod indexers to specify your own.
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:139]:
```suggestion
`default_indexers.enabled`:: (Optional) Enable or disable default pod indexers when you want to specify your own.
```

Comment (dedemorton) [reply libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:139]: my edit is wordier, but it makes it clear that you need to do more than enable or disable the default indexers.

File: libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc [hunk 8]
```diff
@@ -136,7 +136,5 @@ Example:
...
+`default_matchers.enabled`:: (Optional) Enable or disable default pod matchers to specify your own.
```
Comment (dedemorton) [inline libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc:140]:
```suggestion
`default_matchers.enabled`:: (Optional) Enable or disable default pod matchers when you want to specify your own.
```

**Review by dedemorton — APPROVED** [review]

**Pull request merged**